go.mod

```diff
@@ -126,6 +124,7 @@ require (
 	k8s.io/cli-runtime v0.23.3
 	k8s.io/client-go v0.23.3
 	k8s.io/kubectl v0.23.3
+	k8s.io/utils v0.0.0-20211116205334-6203023598ed
```

Comment:
Where did this come from?

Reply:
`go mod tidy`

Reply:
This dependency was specifically added in commit aaa6ea73dd.

Reply:
i think you need to have `e` submodule checked out to run `go mod tidy`